front/src/actions/integration.js

```diff
@@ -43,7 +43,10 @@ const actions = store => ({
     if (searchKeyword && searchKeyword.length > 0) {
       selectedIntegrations = selectedIntegrations.filter(integration => {
         const { name, description } = integration;
-        return name.toLowerCase().includes(searchKeyword) || description.toLowerCase().includes(searchKeyword);
+        return (
+          name.toLowerCase().includes(searchKeyword.toLowerCase()) ||
```

Comment:
Can you please set `searchKeyword.toLowerCase()` in a common variable (defined between line 43 and 44), to avoid to compute "same" lowercase in the loop.

Reply:
Done (push forced)